.github/workflows/ci.yml

```diff
@@ -465,12 +465,27 @@ jobs:
         run: |
           java -version
           mvn install -Dmaven.test.skip
-      - name: Running samples in CI setup
+      - name: Running samples and service client tests in CI setup
```

Comment:
I enabled service client tests for one platform only, because we already test builder and connection methods on all platforms. But if you think it might be useful to run these tests on multiple platforms, I can place them near DeviceAdvisor tests.